recipes/blosc-hdf5-plugin/cmakelists_use_cf_blosc.patch

```diff
@@ -0,0 +1,106 @@
+--- CMakeLists.txt	2019-01-28 18:06:29.524677281 +0100
```

Comment:
Does any of this want to move upstream or is this sufficiently custom to CF that it is not worth it?

Reply:
It is slightly different. If I have time, I can try to incorporate the changes upstream.